.github/workflows/verify.yml

```diff
@@ -0,0 +1,26 @@
+name: verify
+
+on:
+  pull_request:
+    branches: [ '*' ]
+  push:
+    branches: [ main ]
+  workflow_dispatch:
+
+jobs:
+  verify:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.4.0
+
+      - id: get-version
+        run: echo "::set-output name=CURRENT_VERSION::$(cat Makefile | head -n1|awk -F= '{print $2}')"
```

Comment:
`head -n1` won't retrieve the version number anymore since `SHELL=bash` was added to the Makefile. Recommend replacing it with the below to grab the VERSION regardless of its position

```
run: |
    VERSION=$(grep '^VERSION=' Makefile | awk -F= '{print $2}')
    echo "::set-output name=CURRENT_VERSION::$VERSION"
```

Reply:
Yes thats right. updated the code to grab the `version` from the `makefile` properly.